packages/flutter/lib/src/widgets/parent_child_checkbox.dart

```diff
@@ -0,0 +1,183 @@
+import 'package:flutter/material.dart';
+
+///Main widget of Parent Child Checkbox.
```

Comment:
space after `///`. (here and elsewhere)

The first line should be a one sentence description of what the widget is and not just repeat its name.

Also, the doc needs to be expanded with more details about what this widget does and how to use it. Ideally with some sample code.